BondDemo/ListViewController.swift

```diff
@@ -24,19 +24,20 @@ class ListViewController: UITableViewController {
     
     // create view model
     listViewModel = ListViewModel(githubApi: "https://api.github.com/repositories")
+    dataSource = ObservableArray([listViewModel.repositoryCellViewModels])
     
     // create a bond for table view data source
-    tableViewDataSourceBond = UITableViewDataSourceBond(tableView: self.tableView)
 
     // establish a bond between view model and table view
-    listViewModel.repositoryCellViewModels.map { [unowned self] (viewModel: ListCellViewModel, index: Int) -> ListCellView in
-      let cell = (self.tableView.dequeueReusableCellWithIdentifier("cell", forIndexPath: NSIndexPath(forItem: index, inSection: 0)) as? ListCellView)!
-      viewModel.name ->> cell.nameLabel
-      viewModel.username ->> cell.ownerLabel
-      viewModel.photo ->> cell.avatarImageView
+    dataSource.bindTo(tableView) { (indexPath, dataSource, tableView) -> UITableViewCell in
+      let cell = (self.tableView.dequeueReusableCellWithIdentifier("cell", forIndexPath: indexPath) as? ListCellView)!
+      let viewModel = dataSource[indexPath.section][indexPath.row]
+      viewModel.name.bindTo(cell.nameLabel.bnd_text)
+      viewModel.username.bindTo(cell.ownerLabel.bnd_text)
+      viewModel.photo.bindTo(cell.avatarImageView.bnd_image)
```

Comment:
These bindings should be disposed in cell's `prepareForResue` method. The best way is to create a dispose bag as a property on cell, add the disposables returned by the `bindTo` method to that bag and dispose the bag on `prepareForResue`.